testing/integration/fleetserver_test.go

```diff
@@ -165,4 +163,6 @@ func TestInstallFleetServerBootstrap(t *testing.T) {
 		require.Error(t, err, "uninstall should have failed")
 		require.Containsf(t, string(out), "uninstall must be run from outside the installed path", "expected error string not found in: %s err: %s", out, err)
 	}
+
+	t.Run("Test audit/unenroll", testUninstallAuditUnenroll(ctx, fixture, info))
```

Comment:
This is the regression test for this correct?

Reply:
yes, I've refactored `TestInstallUninstallAudit` to also be able to test the audit/unenroll notification for both a standard agent, and one running fleet-server